### PR TITLE
React to AspNetVNext -> AspNetCore feed name change

### PR DESCRIPTION
**Pull request by pranavkm**

### CODE REVIEW

**Comment by pranavkm** [comment]
cc @JunTaoLuo

File: makefile.shade [hunk 1]
```diff
@@ -309,7 +309,7 @@ var buildTarget = "compile"
             }
         }
 
-        Environment.SetEnvironmentVariable("NUGET_VOLATILE_FEED_ASPNETVNEXT", Path.Combine(coherencePath, "packages-expanded"));
+        Environment.SetEnvironmentVariable("NUGET_VOLATILE_FEED_AspNetCore", Path.Combine(coherencePath, "packages-expanded"));
```
Comment (JunTaoLuo) [inline makefile.shade]: `.NUGET_VOLATILE_FEED_ASPNETCORE`?

**Comment by JunTaoLuo** [comment]
:shipit: after casing :nit: comment

File: makefile.shade (hunk 1 above)
Comment (pranavkm) [inline makefile.shade]: The casing matches what we have in NuGet.configs now - https://github.com/aspnet/Common/blob/dev/NuGet.config#L5.

**Pull request merged**